tests/spec/smoketest-eq-services-spec.js

```diff
@@ -0,0 +1,68 @@
+const AuthorHome = require('../pages/AuthorHome.page');
+const CreateQuestionnairePage = require('../pages/CreateQuestionnaire.page');
+const DesignQuestionnairePage = require('../pages/DesignQuestionnaire.page');
+
+const BASE_URL = process.env.BASE_URL || 'http://localhost:3000';
```

Comment:
You should move this to the `baseUrl` config value.
then that will allow you to only need relative paths.